support-files/mysql.spec.sh

```diff
@@ -124,104 +120,42 @@
 %endif
 
 # ----------------------------------------------------------------------------
-# Packager
+# Distribution support
 # ----------------------------------------------------------------------------
-%if %{undefined mysql_packager}
-%define mysql_packager MySQL Build Team <build@mysql.com>
+
+# Disable post build checks for time being.
+BuildConflicts: post-build-checks rpmlint
```

Comment:
Good.
Outside OBS, "rpmlint" seems not to be run automatically, so it should not be an issue.